README.md

```diff
@@ -159,13 +159,32 @@ reformatting is done in a sane manner.
 one by one, as the code formatters (IDEA or Eclipse) tend to create some formats which can and **SHOULD** be manually
 cleaned up, else the code will end up **LESS** readable.
 
-## Javascript
+## JavaScript
 
-Follows "[Google Javascript Style Guide][javascript style page]" with 2 space indents to conform with Java code. Note that IDEA styles import at a global level such that sonatype-idea.xml will import both Java and Javascript style conventions.
+Follows "[Google JavaScript Style Guide][javascript style page]" with 2 space indents to conform with Java code. Note that IDEA styles import at a global level such that sonatype-idea.xml will import both Java and JavaScript style conventions.

```

Comment:
FYI it turns out there are a few ways in which our JS codestyle does not follow the style-guide.

For instance they define that [there should be no newline](https://google.github.io/styleguide/jsguide.html#formatting-nonempty-blocks) before a `else` or a `catch` or a `while` (for a do-while) but we force it.

I wonder if we should just agree that we don't *exactly* follow the google style. Maybe we only follow their lead with indents?